site/content/docs/1.20/config/request-routing.md

```diff
@@ -192,6 +192,7 @@ The mirror traffic is considered _read only_, any response by the mirror will be
 
 This service can be useful for recording traffic for later replay or for smoke testing new deployments.
 
+Without specifying an integer `weight` for the service, 100% of traffic will be mirrored. A non-zero integer weight <=100 directs the corresponding percentage of traffic to the mirror service. 
```

Comment:
Please make this change in `site/content/docs/main/config/request-routing.md` instead of this file. It'll then get copied to version-specific docs when we release.

Reply:
Removed this change and applied it to correct file